platform/ext/target/nordic_nrf/common/nrf5340/tfm_hal_its_encryption.c

```diff
@@ -0,0 +1,204 @@
+#include "platform/include/tfm_platform_system.h"
```

Comment:
Adding the driver wrapper functions here is not a good option I think mainly for 3 reasons:
1) We don't know if directly accessing the driver wrapper is a, generally speaking, accepted solution.
2) Requiring PSA support for this can be problematic because of the overhead it adds.  Smaller platform might easily find it unacceptable. 
3) More error prone to passing the wrong context in the crypto functions